frontend/src/lib/constants.tsx

```diff
@@ -185,3 +185,6 @@ export const UPGRADE_LINK = (cloud?: boolean): { url: string; target?: '_blank'
 
 export const DOMAIN_REGEX = /^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/
 export const SECURE_URL_REGEX = /^(?:http(s)?:\/\/)[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:\/?#[\]@!\$&'\(\)\*\+,;=.]+$/gi
+
+export const CLOUD_US_URL = 'https://app.posthog.com'
+export const CLOUD_EU_URL = 'https://eu.posthog.com'
```

Comment:
Nit but this could be just the hostname, as we for sure want `https://` always :D
Hardcoding overall seem pretty good as a solution – we're definitely not gonna need service discovery for regions. Suggestion though: this can be more obvious as a `Record<Region, string>` constant rather than separate constants